### PR TITLE
docs: add roadmap and update supported versions

### PR DESCRIPTION
**Pull request by gnomeontherun**
This adds a new roadmap page to the website to give a high level overview of what we're working on. It also updates the correct version details for v5 on support page, and a few minor styling fixes.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by hippee-lee — APPROVED** [review]

File: apps/website/.vuepress/theme/styles/overrides.scss [hunk 1]
```diff
@@ -289,6 +289,10 @@ body .no-clr-ui p {
   //margin-bottom: 0;
 }
 
+[cds-text~='body'] li {
+  color: var(--clr-p1-color);
```
Comment (mathisscott) [inline apps/website/.vuepress/theme/styles/overrides.scss]: is there a `--cds-token` we can use here instead?

**Review by mathisscott — APPROVED** [review]

**Comment by mathisscott** [comment]
@gnomeontherun : What's the status here?

**Comment by gnomeontherun** [comment]
Waiting on design for some input.

File: apps/website/.vuepress/theme/styles/overrides.scss (hunk 1 above)
Comment (gnomeontherun) [reply apps/website/.vuepress/theme/styles/overrides.scss]: I wanted to map this to the paragraphs around it, so this seems like the best one?

Comment (mathisscott) [reply apps/website/.vuepress/theme/styles/overrides.scss]: The problem here is that the `--clr-p?-color` custom properties are from Clarity UI. I think we want to limit our dependency there if possible. If you want to assign the color with an exact match in the Core design tokens, the color you want is `--cds-global-color-gray-900`. I did the digging to find what that p1's color value was and `gray-900` is a 100% match.

Comment (mathisscott) [inline apps/website/.vuepress/theme/styles/overrides.scss]: Update to use `--cds-global-color-gray-900`

**Review by mathisscott — CHANGES_REQUESTED** [review]

File: apps/website/.vuepress/theme/styles/overrides.scss (hunk 1 above)
Comment (gnomeontherun) [reply apps/website/.vuepress/theme/styles/overrides.scss]: 700 is the correct color.

**Comment by gnomeontherun** [comment]
I also added a Clarity Angular <-> Angular compatibility list.

**Review by mathisscott — APPROVED** [review]

**Pull request merged**